### PR TITLE
Feature/redis

### PR DESCRIPTION
**Pull request by juliantopley**

### CODE REVIEW

File: _docs/applications/postgres.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: guide
-title: StorageOS Docs - PostgreSQL
+title: PostgreSQL with StorageOS
```
Comment (oicheryl) [inline _docs/applications/postgres.md]: Don't change this, this is displayed in the tab title bar, so all guides should follow the same format of "StorageOS Docs - ".

Comment (juliantopley) [inline _docs/applications/postgres.md]: This is not consistent - I'll change the other pages under applications to reflect this as its just MS SQL and Postgres currently.

File: _guides/004-applications.md [hunk 2]
```diff
@@ -20,8 +20,5 @@ sections:
     title: Redis
```
Comment (oicheryl) [inline _guides/004-applications.md]: StorageOS - Redis

Comment (juliantopley) [inline _guides/004-applications.md]: Thats what I have:

layout: guide
title: StorageOS Docs - Redis
anchor: applications
module: applications/redis

Comment (juliantopley) [inline _guides/004-applications.md]: What do you want to change?

**Pull request merged**